validators/terravaloper1xapwn8pnqu4l2v3wmpslq2zxr5v5jwjks20dhu/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-Moniker: Ninja Node🥷🏼⚡(0% fee) | LUNCninja.com
+Moniker: Ninja Node (0% fee) | LUNCninja.com
```

Comment:
Your moniker does seem to include the emojis in the [explorer](https://finder.terra.money/classic/validator/terravaloper1xapwn8pnqu4l2v3wmpslq2zxr5v5jwjks20dhu).

Are you sure you want to change the name here?

Reply:
Hey @terran6. I only really wanted to change the file name since I misspelled README. 
I don't want to change the moniker.
However I found this error message in the file: "Error in user YAML: (<unknown>): found character that cannot start any token while scanning for the next token at line 4 column 11" so I thought it might have been related to emojis in the moniker. Feel free to add them back if it is not the cause of the error message.

Reply:
@LUNCNinjaNode Ah I see the filename change.  Okay, I can merge this and hopefully, the error will resolve.  If it does not, you can feel free to add the emojis back in if you prefer.  Thanks!

Reply:
Thanks! I see the error is still present. It seems to have something to do with tabs vs spaces. I'll have to try again

Reply:
However, I got the desired outcome, so I wont change anyting. Thanks for your help :)